_docs/quickstarts/scanner/default-rules.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Default Rules
+order: 106
+---
+
+# {{ page.title }}
+
+Forseti Scanner is preconfigured with some default rules. When any of these rules' conditions are met, 
+Scanner will create a [violation]({% link _docs/quickstarts/scanner/policy-violations.md %}).
+
+* BigQuery
+  * Datasets should not be public.
+  * Datasets should not be accessible by users who are @gmail.com.
+  * Datasets should not be accessible by groups who are @gmail.com.
+ 
+* Cloud Storage
+  * Buckets should not be publicly accessible (“AllUsers”).
```

Comment:
For my clarification, is this checking the bucket acl, or the default object acl, or both?